config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  get '/tasks', to: 'tasks#index', as: 'tasks'
```

Comment:
This is how you add a root path so that localhost:3000 points to the task index page.

```suggestion
  root to: 'tasks#index'
```